packages/calcite-components/src/components/split-button/split-button.scss

```diff
@@ -98,6 +98,7 @@
 :host([appearance="outline"]) {
   .divider-container {
     border-block: 1px solid var(--calcite-split-button-divider);
+    background-color: initial;
```

Comment:
The divider container's [background-color is set via custom CSS prop](https://github.com/Esri/calcite-design-system/blob/josercarcamo/8142-split-button-consistent-dividers/packages/calcite-components/src/components/split-button/split-button.scss#L89), so could you try updating [`--calcite-split-button-background`](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/components/split-button/split-button.scss#L55-L61) instead of overriding here? It looks like it should be `transparent` for both `outline` and `outline-fill` appearance types.